ext/smoke/packages/steps/setup-agent.sh

```diff
@@ -0,0 +1,81 @@
+#!/bin/bash
+
+set -e
```

Comment:
This could also be re-used, aside from the installation of the agent package step (repo vs. raw RPM). Maybe an extra arg. could be added to differentiate if we're installing the raw package or from the repos and case it on that? Should be OK since we don't have any optional arguments here.

Reply:
Do we still need this? I see you have it at the top level.